feature-flags/src/redis.rs

```diff
@@ -40,38 +43,31 @@ impl Client for RedisClient {
 
         Ok(fut?)
     }
-}
 
-// TODO: Find if there's a better way around this.
-#[derive(Clone)]
-pub struct MockRedisClient {
-    zrangebyscore_ret: Vec<String>,
-}
+    async fn get(&self, k: String) -> Result<String> {
```

Comment:
TODO: because rust exposes all string handling, double check if something very bad will happen if I have a utf-16 character in the team name

Reply:
eh will come back to this, will remove team name from consideration for now, will handle this on string matching properties.